src/modules/job-exec/exec_config.c

```diff
@@ -114,12 +120,13 @@ int config_init (flux_t *h, int argc, char **argv)
         return -1;
     }
 
-    /*  Check configuration for exec.service */
+    /*  Check configuration for exec.service and exec.service-override */
     if (flux_conf_unpack (flux_get_conf (h),
                           &err,
-                          "{s?{s?s}}",
+                          "{s?{s?s s?b}}",
                           "exec",
-                            "service", &exec_service) < 0) {
+                            "service", &exec_service,
+                            "service-override", &exec_service_override) < 0) {
```

Comment:
should this be documented somewhere?  although skimming comments at top of files in `job-exec` i'm not sure where.

Reply:
Well maybe I ought to just go ahead and document all the configuration in man pages.